### PR TITLE
fix: prettier

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
no such thing as es6

### CODE REVIEW

File: .prettierrc.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
-    "trailingComma": "es6",
```
Comment (HarryET) [inline .prettierrc.json:2]: My bad, sorry. Replace with es5 or check docs for latest version that matches with the tsconfig

**Review by HarryET — DISMISSED** [review]

File: .prettierrc.json (hunk 1 above)
Comment (Isaiah-Hamilton) [reply .prettierrc.json:2]: es5 is the latest version

**Comment by Isaiah-Hamilton** [comment]
Also added a prettier code check.

File: .prettierrc.json (hunk 1 above)
Comment (VictorPeralta) [reply .prettierrc.json:2]: We can remove it altogether, it's enabled by default

**Review by VictorPeralta — DISMISSED** [review]

**Review by VictorPeralta — APPROVED** [review]

**Pull request merged**